tests/components/vesync/test_common.py

```diff
@@ -0,0 +1,166 @@
+"""Tests for VeSync common utilities."""
```

Comment:
It is recommended that utilities are not tested "on their own".
They should be covered instead by the normal init or platform tests.

Reply:
I'm not sure I understand why that would be the recommendation.  The purpose of testing the utilities on their own is to certify that they work as expected with a range of inputs.  This prevents contaminating platform tests with variations to ensure the utilities are functioning correctly.

Reply:
As an added note, I know that I have multiple utility methods that i will be adding as I increase the functionality of this integration, what you see now is just the part that applies today.

Reply:
removed the non-platform state test code